pykrige/ok.py

```diff
@@ -189,7 +189,7 @@ class OrdinaryKriging:
     def __init__(self, x, y, z, variogram_model='linear', variogram_parameters=None,
                  variogram_function=None, nlags=6, weight=False, anisotropy_scaling=1.0,
                  anisotropy_angle=0.0, verbose=False, enable_plotting=False,
-                 enable_statistics=False):
+                 enable_statistics=False, coordinates_type='euclidean'):

```

Comment:
Need to add an if test and raise an error is `corrdinates_type` is not one of the 2 acceptable values.